### PR TITLE
WT-6722 Review function names in history store module

### PR DESCRIPTION
**Pull request by hbokhari**
There is relevant discussion in Jira ticket on the naming scheme that I am following for this PR.

### CODE REVIEW

File: src/history/hs_cursor.c [hunk 1]
```diff
@@ -9,11 +9,11 @@
 #include "wt_internal.h"
 
 /*
- * __wt_hs_row_search --
+ * __wti_hs_row_search --
```
Comment (agorrod) [inline src/history/hs_cursor.c:12]: If there is only one place where the `\_\_wti_` prefix is relevant, I don't think we should introduce this new protocol, as we have different solutions to the problem (as touched on in the ticket)

Comment (hbokhari) [reply src/history/hs_cursor.c:12]: I agree with you on this one. Most of the function are called from both within the history store subsystem and outside the subsystem and that's why we use `__wt_` prefix. I am not sure how this situation is for other sub-systems, but definitely a single intra-module function in history store module doesn't warrant a new naming prefix.

**Review by quchenhao — APPROVED** [review]
lgtm

**Review by agorrod — APPROVED** [review]
LGTM

**Review by bvpvamsikrishna — APPROVED** [review]
lgtm

**Pull request merged**